pm/stub.go

```diff
@@ -0,0 +1,137 @@
+package pm
+
+import (
+	"fmt"
+	"math/big"
+
+	ethcommon "github.com/ethereum/go-ethereum/common"
+)
+
+type stubSigVerifier struct {
+	shouldVerify bool
+}
+
+func (sv *stubSigVerifier) SetShouldVerify(shouldVerify bool) {
```

Comment:
I think this function name is a bit confusing. Perhaps changing it to `SetVerifyResult` is a bit clearer?